tests/Unit/AbstractUuid4TypeTest.php

```diff
@@ -2,26 +2,26 @@
 
 namespace Fortuneglobe\Types\Tests\Unit;
 
-use Fortuneglobe\Types\Exceptions\InvalidUuidValueException;
+use Fortuneglobe\Types\AbstractUuid4Type;
+use Fortuneglobe\Types\Exceptions\InvalidUuid4ValueException;
 use Fortuneglobe\Types\Interfaces\RepresentsScalarValue;
-use Fortuneglobe\Types\Interfaces\RepresentsUuidValue;
-use Fortuneglobe\Types\Tests\Unit\Fixtures\TestUuid;
-use Fortuneglobe\Types\UuidType;
+use Fortuneglobe\Types\Interfaces\RepresentsUuid4Value;
+use Fortuneglobe\Types\Tests\Unit\Fixtures\TestUuid4;
 use PHPUnit\Framework\TestCase;
 
 /**
  * Class UuidTypeTest
  * @package Fortuneglobe\Types\Tests\Unit
  */
-final class UuidTypeTest extends TestCase
```

Comment:
It's `AbstractUuid4TypeTest`.

Reply:
Fixed in 70fe97374eeed55e6dce6d5d1872fd2f9ec9974c